doc/sample/getiteminfo.txt

```diff
@@ -34,5 +34,37 @@ prontera,156,179,6	script	test_getiteminfo	4_F_KAFRA1,{
 	mesf("View ID: %d", getiteminfo(.@value, ITEMINFO_VIEWID));
 	mesf("MATK: %d", getiteminfo(.@value, ITEMINFO_MATK));
 	mesf("View Sprite: %d", getiteminfo(.@value, ITEMINFO_VIEWSPRITE));
+	.@trade = getiteminfo(.@value, ITEMINFO_TRADE);
+	.@trade$ = "";
```

Comment:
probably better move this into a global function kinda like what we do for `ITEMINFO_LOC` with `F_GetArmorType`